boards/Kconfig

```diff
@@ -3389,6 +3389,13 @@ config BOARDCTL_UNIQUEKEY_SIZE
 		Provides the size of the memory buffer that must be provided by the
 		caller of board_uniquekey() in which to receive the board unique KEY.
 
+config BOARDCTL_SWITCH_BOOT
+	bool "Switch Boot A/B System"
+	default n
+	---help---
+		Enables support for the BOARDIOC_SWITCH_BOOT boardctl() command.
+		Switch to the updated or the specified Boot A/B system.
+
```

Comment:
This is to vague. Please add more information:

This API can be used to change the system boot behavior.  For instance, once a firmware updated has completed successfully this boardctl can be used modify FLASH bank selection.

Reply:
@xiaoxiang781216 The salient points from [comment](https://github.com/apache/incubator-nuttx/pull/4191#issuecomment-883941665) should be added to either the Kconfig.

Reply:
@davids5 @xiaoxiang781216 I have updated the patch accordingly. Pls help review, thanks.